code/modules/mob/living/carbon/human/life.dm

```diff
@@ -812,7 +814,7 @@ var/global/list/brutefireloss_overlays = list("1" = image("icon" = 'icons/mob/sc
 
 	proc/handle_chemicals_in_body()
 
-		if(reagents && !(species.flags & IS_SYNTHETIC)) //Synths don't process reagents.
+		if(reagents) //Synths don't process reagents. // fuck you they do now - Iamgoofball

```

Comment:
everyone gets to be on an even playing field when it comes to chemicals

none of your bullshit snowflake in my chemistry code